README.md

```diff
@@ -333,7 +357,20 @@ To display the logs of the check results, run
 docker-compose logs prereqs-checker
 ```
 
+### DX Development with Visual Studio Code & Docker-Compose  
 
+Please check the course content:  
+[HDX-DEV-300 HCL Digital Experience for Developers (Advanced)](https://hclsoftwareu.hcltechsw.com/courses/course/hdx-dev-300-dx-developer-advanced)  
 
+### Best practice  
 
-
+- Always use `docker-compose up` command to start your environment
+- Make sure that the installApps.sh / installApps.bat script is already completed, before accessing the environment.
+  Only then the additional extentions like Content Composer and DAM will be available!
+- Try to avoid using volumes (at best don't use volumes at all to get the best spead)
+- Never restart the dx-core container (If needed, just stop/start the server directly in the dx-core container)
+- Add shared libraries into folder `/opt/HCL/wp_profile/PortalServer/sharedLibrary/` of the dx-core container
+- In the Web-Browser:  
+  - Access your Portal environment with URL: `http://<hostname>/wps/portal`.  
```

Comment:
```suggestion
  - Access your Portal environment with URL: `http://<hostname>/wps/portal` or `https://<hostname>/wps/portal`.  
```

Reply:
Hi Thomas, https://<hostname>/wps/portal is not working at all with the defautl settings in the dx.yaml! I would not mention it!